src/posts/introducing-xradar/index.md

```diff
@@ -0,0 +1,208 @@
+
+## A typical Radar Data Analysis Workflow with `Xradar`
+
+1. Install it in your environment using either
+   - [conda](https://anaconda.org/conda-forge/xradar) (`conda install -c conda-forge xradar`)
+   - or [pip](https://pypi.org/project/xradar/) (`pip install xradar`)
+2. Open up a notebook and use the imports shown below!
+
+```python
+import os
+import xarray as xr
+import xradar as xd
+
+# retrieval of data files
+from open_radar_data import DATASETS
+```
+
+### Fetch and Open a Single Sweep of CfRadial1 data
+
+Within xradar, we have a collection of **xarray backends**, which enable us to read a variety of radar data formats using xarray! In this case, we are reading _cfradial1_ data, which follows the data structure described above.
+
+We can use the xarray API directly, specifying we want the first sweep (counting from 0), `sweep_0`. This returns an xarray dataset!
+
+```python
+filename = DATASETS.fetch("cfrad.20080604_002217_000_SPOL_v36_SUR.nc")
+ds = xr.open_dataset(filename, group="sweep_0", engine="cfradial1")
+ds
+```
+
+<iframe
+  width='100%'
+  height='500'
+  src='/posts/introducing-xradar/out01.html'
+  frameborder='0'
+  allowfullscreen
+></iframe>
+
+### Open an Entire Volume of Cfradial1 data
+
+Often times, we want the **_entire volume_**, including all of the individual sweeps. Since the coordiantes vary across each sweep, and there are additional metadata fields that do not conform to the typical `xarray.Dataset` data structure, we use the [`xarray.DataTree`](https://github.com/xarray-contrib/datatree) data model. We call this tree of radar data a **radar object** since it represents the entire set of radar data.
+
+```python
+radar = xd.io.open_cfradial1_datatree(filename)
+radar
+```
+
+<iframe
+  width='100%'
+  height='500'
+  src='/posts/introducing-xradar/out02.html'
+  frameborder='0'
+  allowfullscreen
+></iframe>
+
+### Georeference data
+
+By default, the coordinates are in antenna coordinates, with limited geographic information. The only latitude/longitude information included in the file is the location of the radar, with the individual **gate latitude and longitude** information missing. Our first step to plotting this data and performing anlaysis is georeferencing, or adding additional geographic information to the coordinates.
+
+Here, we use the **_Azimuthal Equidistant_** projection, with the x/y/z values representing the distance east/west, north/south, above/below the radar respectively. The georeferencing can be performed on the core **radar** object using the **georeference accessor**. An example is provided below.
+
+Notice how after we apply this georeferencing, values of x, y, and z are added to each of the sweeps!
+
+```python
+radar = radar.xradar.georeference()
+radar['sweep_0']
+```
+
+<iframe
+  width='100%'
+  height='500'
+  src='/posts/introducing-xradar/out03.html'
+  frameborder='0'
+  allowfullscreen
+></iframe>
```

Comment:
This isn't a bad solution. @andersy005 - do you think we can make this less painful by following a similar approach to Pangeo-forge: 

https://github.com/pangeo-forge/pangeo-forge.org/blob/97efc8292d04cf973c60020588b40c140de38557/src/components/dataset/repr.js#LL41C14-L41C37

Reply:
I did not try harder, but one remaining problem is the fixed height. I could not yet get it to expand the window on display of variables for instance.